paasta_tools/contrib/paasta_update_soa_memcpu.py

```diff
@@ -98,66 +116,109 @@ def get_reviewers(filename):
 
     instdict = data[instance]
-    if mem:
-        instdict['mem'] = mem
-    else:
-        instdict['cpus'] = cpu
+    instdict['cpus'] = cpu
     out = yaml.round_trip_dump(data, width=10000)
 
     with open(filename, 'w') as fi:
         fi.write(out)
 
 
+def create_jira_ticket(serv, creds, description):
+    creds = creds.split(':')
+    options = {'server': 'https://jira.yelpcorp.com'}
+    jira_cli = JIRA(options=options, basic_auth=(creds[0], creds[1]))
+    jira_ticket = {}
+    # Sometimes the project name doesn't match a JIRA project
+    try:
+        jira_ticket = {
+            'project': {'key': serv['project']},
+            'issuetype': {'name': 'Improvement'},
+            'description': description,
+            'labels': ['perf-watching', 'paasta-rightsizer'],
+            'summary': "{s} {c} {i} may be {o}provisioned".format(
+                s=serv['service'],
+                i=serv['instance'],
+                c=serv['cluster'],
+                o=serv['over'],
+            ),
+        }
+        tick = jira_cli.create_issue(fields=jira_ticket)
+    except Exception:
+        jira_ticket = {
+            'project': {'key': 'PERF'},
+            'issuetype': {'name': 'Improvement'},
+            'description': description,
+            'summary': "{s} {c} {i} may be {o}provisioned".format(
+                s=serv['service'],
+                i=serv['instance'],
+                c=serv['cluster'],
+                o=serv['over'],
+            ),
+            'labels': ['perf-watching', 'paasta-rightsizer'],
+        }
+        tick = jira_cli.create_issue(fields=jira_ticket)
+    return tick.key
+
+
 def main(argv=None):
     args = parse_args(argv)
-    mem = args.mem
-    cpu = args.cpu
-    instance = args.instance
-    service = args.service
-    cluster = args.cluster
-    ticket = args.ticket
-
-    filename = '{}/marathon-{}.yaml'.format(service, cluster)
-    memcpu = ''
-    if mem:
-        memcpu = 'mem'
-        mem = int(mem)
-    elif cpu:
-        memcpu = 'cpu'
-        cpu = int(cpu)
-    else:
-        print('please specify either mem or cpu')
-        sys.exit(2)
-
-    with in_tempdir():
-        clone(ticket)
-        edit_soa_configs(filename, instance, mem, cpu)
-        commit(filename, memcpu)
-        review(filename)
+    services_to_update = get_perf_data(args.splunk_creds, args.file_splunk)
+
+    for serv in services_to_update:
+        filename = '{}/{}.yaml'.format(serv['service'], serv['cluster'])
+        cpus = float(serv['cpus'])
+        over = 'over'
+        if cpus > float(serv['old_cpus']):
+            over = 'under'
+
+        serv['over'] = over
+        ticket_desc = (
+            "We suspect that {s} {c} {i} may be {o}-provisioned"
+            " as of {d}.\n- Dashboard: y/{o}provisioned\n- Service"
+            " owner: {n}\n- Estimated monthly excess cost: {m}"
+            "\n- Runbook: y/rb-provisioning-alert"
```

Comment:
This needs to have maximum context. Can you add the cpu data too? 

    it originally had N cpus, but we think it only needs to have M cpus. Here is a graph to the dashboard: